### PR TITLE
[NUI][TCSACR-343] Add guide for ScrollableBase

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Change Description ###

Add guide for ScrollableBase

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-343

### CODE REVIEW

**Review by elishateng — APPROVED** [review]
LGTM

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by huayongxu — APPROVED** [review]
LGTM.

**Comment by mijin85cho** [comment]
Hello, @surbhig12.
As @annie-samsung is absent now, would you please assign the reviewer instead?

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@zhouleonlei request you to incorporate the comments.
Thanks

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# ScrollableBase
+
+ScrollableBase is a common component which support scrolling. Several views can be added to ScrollableBase.
+User can use mouse or finger to drag it on the touch screen.
+
+![ScrollableBase](./media/ScrollableBase.png)
+
+## Create with property
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md:8]: @nawab-samsung Please check the consistency with other pages. I think keeping the current status would be Okay.

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        }
+        else
+        {
+            items[i].BackgroundColor = Color.Cyan;
+        }
+        scrollableBase.Add(items[i]);
+    }
+    ```
+
+Following output is generated when the ScrollableBase is created using property:
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md:49]: @nawab-samsung I think you can check other guide pages also to check the consistency.. I think we can keep the original.

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md:8]: @mijin85cho sure will check. 
Thanks

[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md:8]: @mijin85cho I agree with you, this is ok, so as to maintain consistency. 
Thanks

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md:49]: @mijin85cho I agree with you, this is ok, so as to maintain consistency.
Thanks

**Comment by nawab-samsung** [comment]
@zhouleonlei can you please incorporate the comments?

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# ScrollableBase
+
+ScrollableBase is a common component which support scrolling. Several views can be added to ScrollableBase.
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+User can use mouse or finger to drag it on the touch screen.
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        }
+        scrollableBase.Add(items[i]);
+    }
+    ```
+
+Following output is generated when the ScrollableBase is created using property:
+
+![ScrollableBaseProperty](./media/ScrollableBase.png)
+
+## Responding to scroll drag events
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
+# ScrollableBase
+
+ScrollableBase is a common component which support scrolling. Several views can be added to ScrollableBase.
+User can use mouse or finger to drag it on the touch screen.
+
+![ScrollableBase](./media/ScrollableBase.png)
+
+## Create with property
+
+To create a ScrollableBase using property, follow these steps:
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+Following output is generated when the ScrollableBase is created using property:
+
+![ScrollableBaseProperty](./media/ScrollableBase.png)
+
+## Responding to scroll drag events
+
+When you drag a ScrollableBase, the ScrollableBase instance receives a scroll drag started event.
+And after you stop dragging, the ScrollableBase instance receives a scroll drag ended event.
+You can declare the events handlers as follows:
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    // Do something in response to scroll drag started
+}
+
+private void ScrollDragEnded(object sender, ScrollEventArgs e)
+{
+    // Do something in response to scroll drag ended
+}
+```
+
+Following output is generated when the ScrollableBase is being dragged:
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+![ScrollableBaseDrag](./media/ScrollableBase.gif)
+
+## Related Information
```
[the remark this answers is not in the document]
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

**Comment by zhouleonlei** [comment]
> @zhouleonlei can you please incorporate the comments?

Finished

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+
+![ScrollableBase](./media/ScrollableBase.png)
+
+## Create with property
+
+To create a ScrollableBase using various properties, follow these steps:
+
+1. Create ScrollableBase using the default constructor:
+
+    ```cs
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md:14]: As per our style guide, for language identification : 

Global: Please replace cs with csharp
Reference:  https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#supported-languages

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@zhouleonlei Just once thing needs a fix, then I will approve the PR.

File: docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md (hunk 10 above)
Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md]: Done

**Review by nawab-samsung — APPROVED** [review]
@zhouleonlei thanks for incorporating the comments

**Review by mijin85cho — APPROVED** [review]
Thanks for the updates and reviews.

**Pull request merged**